### PR TITLE
Added weather api 

### PR DESCRIPTION
**Pull request by Pgandhi21**
fetching weather data

### CODE REVIEW

File: assets/javascript/datapage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    fetch(latLonURL)
+        .then(response =>response.json())
+        .then(function(data) {
+            console.log(data);
+            cityName = data.name;
+            lat = data.coord.lat;
+            lon = data.coord.lon;
+            weatherUrl(lat, lon, cityName);
+            console.log(lat);
+            console.log(lon);
```
Comment (blackman3178) [inline assets/javascript/datapage.js]: var latLong = {
lat : lat,
long: long
}

File: assets/javascript/datapage.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        .then(response =>response.json())
+        .then(function(data) {
+            console.log(data);
+            cityName = data.name;
+            lat = data.coord.lat;
+            lon = data.coord.lon;
+            weatherUrl(lat, lon, cityName);
+            console.log(lat);
+            console.log(lon);     
+    });
```
Comment (blackman3178) [inline assets/javascript/datapage.js:21]: return latLong

**Review by blackman3178 — APPROVED** [review]
great Job!!

**Review by blackman3178 — APPROVED** [review]

**Review by abrahamin — APPROVED** [review]
Good job!

**Pull request merged**